src/index.html

```diff
@@ -1,14 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <a
+        href="https://www.nasa.gov/"
+        alt=""
+        class="logo"
+      >
+        <img src="/images/logo.png" alt="">
+      </a>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="nav_item">
+            <a href="https://www.nasa.gov/" class="menuItem is-active">apple</a>
```

Comment:
I do not know is it new or I missed it previous time but use kebab-case for class names not camelCase